src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,111 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int INIT_SIZE = 10;
+    private static Object[] array = new Object[INIT_SIZE];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
int field is 0 by default